_data/categories.yml

```diff
@@ -1,9 +1,11 @@
-order:
+order: 
+  - name: "API Reference"
```

Comment:
Should be specific to Angelscript imo. Maybe something like `AngelScript Hammer API`? That might be a little wordy to fit on one line.

Reply:
I previewed `AngelScript Hammer API` locally and it looks okay --
![image](https://user-images.githubusercontent.com/7349732/145696962-38283c71-21fb-4156-8ab0-5be6e357abca.png)

I handled that rename in [`1ff4b97` (#231)](https://github.com/momentum-mod/docs/pull/231/commits/1ff4b97ffbf6471f493c418f65d3fba47cefbe4a) and updated the directory/category renaming (`api` -> `angelscript`) in [`5fd014f` (#231)](https://github.com/momentum-mod/docs/pull/231/commits/5fd014fc621dfc6497855316326b3f695807578a)

If the post naming is a little redundant under that new section, I can update that as well.